tests/e2e-playwright/Makefile

```diff
@@ -117,13 +117,17 @@ CLASSIC_TIP_INPUT_FILE := .e2e-playwright-classictip-env.txt
 $(SLEEPERS_INPUT_FILE) $(JUPYTER_LAB_INPUT_FILE) $(CLASSIC_TIP_INPUT_FILE) $(S4L_INPUT_FILE):
 	@read -p "Enter your product URL: " PRODUCT_URL; \
 	read -p "Is the product billable [y/n]: " BILLABLE; \
+	read -p "Is the product lite [y/n]: " IS_LITE; \
 	read -p "Is the test running in autoscaled deployment [y/n]: " AUTOSCALED; \
 	read -p "Enter your username: " USER_NAME; \
 	read -s -p "Enter your password: " PASSWORD; echo ""; \
 	echo "--product-url=$$PRODUCT_URL --user-name=$$USER_NAME --password=$$PASSWORD" > $@; \
 	if [ "$$BILLABLE" = "y" ]; then \
 		echo "--product-billable" >> $@; \
 	fi; \
+	if [ "$$IS_LITE" = "y" ]; then \
+		echo "--product-lite" >> $@; \
```

Comment:
This might be a bit confusing. it's not used in s4l-lite, it's only for tip-lite, therefore I would rename it to ``product-tip-lite``

Reply:
for the moment, I prefer to have it as a generic flag so that we can reuse workflows targeting "full" versions